.github/workflows/deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: deploy kikeriki to docker hub
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v3
+    - name: Set up JDK 1.17
+      uses: actions/setup-java@v3
+      with:
+        java-version: 17.0.4+1
+        distribution: zulu
+    - name: Build the project with Maven
+      run: ./mvnw --batch-mode -update-snapshots package
+    - name: Login to the Docker registry
+      run:  docker login --username ${{ secrets.DOCKER_HUB_USER }} --password ${{ secrets.DOCKER_HUB_PASS }}
+    - name: Build the Docker image
+      run:  docker build -t dancier/kikeriki:$(git rev-parse --short "$GITHUB_SHA") -t dancier/kikeriki:latest .
+    - name: Publish the Docker image on hub.docker.com
+      run:  docker push -a dancier/kikeriki
```

Comment:
Added a workflow to deploy to docker hub

Reply:
Intresting :)
O.k., there, you need the maven-wrapper.
I removed it, because I don't like it.
I don't think, that it is really necessary to always build with the same version locally.
Yes, it might break because a version change for Maven - but the build, that must not break because of a version change is the one in the build-pipeline.
But it is o.k. for me...

Reply:
And the build system would use the same as on your machine then ;-)